randomgen/generator.pyx

```diff
@@ -655,23 +660,23 @@ cdef class RandomGenerator:
             raise TypeError('Unsupported dtype "%s" for randint' % key)
 
         if key == 'int32':
-            ret =  _rand_int32(low, high, size, self._brng, self.lock)
+            ret =  _rand_int32(low, high, size, self._brng, self.lock, use_masked_generator)
```

Comment:
Not sure if too pedantic, but should use_maked_generator come after size and before brng and lock, for consistency with `def` (and most others)

Reply:
I think you're right, it should be after size.